commands/instance/instance.go

```diff
@@ -18,6 +18,8 @@
 package instance
 
 import (
+	"github.com/apache/skywalking-cli/graphql/client"
+	"github.com/apache/skywalking-cli/logger"
 	"github.com/urfave/cli"
```

Comment:
Failed because of `import` order, run `$(go env GOPATH)/bin/golangci-lint run -v --fix ./...` locally to fix

Reply:
@kezhenxu94 Thanks. It's fixed.